modules/local/csvtk_concat.nf

```diff
@@ -1,7 +1,7 @@
 process CSVTK_CONCAT {
```

Comment:
Any reason why this and split are local modules not the official nf-core ones? If you just want the sort command I would just patch the module with `nf-core modules patch` rather than having an entirely separate 'cusotm' local one

Reply:
Sounds good!